esp-hal/Cargo.toml

```diff
@@ -152,6 +152,9 @@ octal-psram = []
 # This feature is intended for testing; you probably don't want to enable it:
 ci = ["defmt", "bluetooth"]
 
+# Internal feature set by esp-wifi
+__include_rom_functions = []
```

Comment:
Can you move this next to `__esp_hal_embassy`? Let's group the internal features - also not necesary to document IMHO (`__esp_hal_embassy` doesn't show up in the docs and I think that's all right)